### ISSUE TITLE
Running 2 versions of s3fs on same server

### ISSUE THREAD

**Issue by chrisgo** [s3fs-fuse/s3fs-fuse#408]
Using fstab, debian 8

Due to some compatibility issues (currently running 1.61), I need to run 2 version of s3fs - the 1.61 version and s3fs "latest".  Is this possible?  How to use this with fstab?

Compatibility issues here:  https://github.com/s3fs-fuse/s3fs-fuse/issues/380

**Comment by ggtakec** [comment]
@chrisgo 
Maybe I think you can run both s3fs version. If you use cache for s3fs, you will need to divide the value of the use_cache. And you will also need to watch out for other options.

I think you should try the following procedure.
1) You are to make sure that each version of s3fs starts alone
2) To start both versions at the same time

However, what is the compatibility issues that you say to?
s3fs is maintaining backward compatibility as much as possible, then please tell us that you have a problem with the latest version.
I think that there may be other ways.

Thanks in advance for your help.

**Comment by chrisgo** [comment]
@ggtakec thank you for your response

The main issue that I have found is that the old version (1.61) creates extra "0 byte" files when you create a folder structure inside the bucket

So, for example:  `bucket-name/subfolder/file.txt`

would get created as 2 files instead of just file.txt 

```
[file icon] bucket-name/subfolder
[directory icon] bucket-name/subfolder/file.txt
```
In the Properties tab (AWS console)
The `subfolder` entry above will be called `Object: subfolder`
In the new s3fs (and other tools), `subfolder` will actually say `Folder: subfolder`
file.txt is `File: file.txt`

So my current problem right now is to try to find a way to migrate these somehow to the new s3fs binary.  However, since all my current servers (not too many < 20 all mounting the same bucket) are running the old s3fs 1.61, I have to do a bunch of testing to see what happens if I just upgrade the command line -- will everything work?   The last time I tested it (maybe a year ago), some of the permissions got strange (code stopped being able to write to the mounts) so I had to back out and keep using the old 1.61.  


Thanks,
Chris
